packages/disputer/test/Disputer.js

```diff
@@ -1,31 +1,25 @@
+const { toWei, toBN, utf8ToHex, padRight } = web3.utils;
```

Comment:
this file is somewhat hard to review as a lot of indenting happened through. the making change is that there is a new unit test at the end that is perpetual and funding rate spesific. The rest of the logic is pretty much the same, simply adding the muliversion test helpers through to ensure that existing tests work with all contract versions.

Reply:
Do all of the bot tests get run by CI?

Reply:
yup.